tin/apps/assignments/models.py

```diff
@@ -172,6 +173,16 @@ class Assignment(models.Model):
         validators=[MinValueValidator(10)],
     )
 
+    use_submission_cap = models.BooleanField(default=False)
+    submission_cap = models.PositiveSmallIntegerField(
+        null=True,
+        validators=[MinValueValidator(1)],
+    )
+    submission_cap_after_due = models.PositiveSmallIntegerField(
+        null=True,
+        validators=[MinValueValidator(1)],
```

Comment:
Why can't the after-due-date-cap be zero? Seems like a logical use of the feature...